src/Core/tests/DeviceTests/Handlers/Button/ButtonHandlerTests.iOS.cs

```diff
@@ -86,7 +86,8 @@ Task PerformClick(IButton button)
 		{
 			var button = GetNativeButton(buttonHandler);
 
-			var attributedText = button.TitleLabel.AttributedText;
+			//var attributedText = button.TitleLabel.AttributedText;
+			var attributedText = button.GetAttributedTitle(UIControlState.Normal);
```

Comment:
The changes looks nice, and we already have tests validating the CharacterSpacing property but could we add a device test where we get the width of the native button before updating the CharacterSpacing and after setting it and check that the updated value is greater than the initial one?